### ISSUE TITLE
Nested Relay Pagination `last` Filter returns empty list of edges

### ISSUE THREAD

**Issue by rsomani95** [MrThearMan/graphene-django-query-optimizer#75]
### I have read the docs thoroughly before making this bug report.

- [X] Yes

### I have read through other open issues, and my issue is not a duplicate.

- [X] Yes

### What version of the library you are using?

0.2.8

### Which python version are you using?

3.12

### What operating system are you on?

Mac

### Description

When querying the `last` filter on a nested relay node, like so:

```graphql
query {
  pagedHousingCompanies {
    edges {
      node {
        name
        developers(last: 1) {
          edges {
            node {
              name
            }
          }
        }
      }
    }
  }
}

```

empty sets are returned for all edges.

If I don't include any pagination filter, I can see multiple `developers` related to each housing company. The `first` filter works correctly too.

**Comment by MrThearMan** [comment]
This is a limitation of how the queryset slicing currently works for prefetched querysets.

What currently happens is that we simply use the `RELAY_CONNECTION_MAX_LIMIT` setting to limit the querysets, so when you query for `last:1`, the queryset is limited with `queryset[99:100]`, which in most cases will be an empty queryset.

Remember that for `totalCount`, its costly to calculate the actual number of items in each partition. In order for the last argument to work propertly, it needs to know the last possible item in each partition, so we would need to calculate the `totalCount` here as well. 

Furthermore, the function currently calculating the slice is implemented in python ([calculate_queryset_slicing](https://github.com/MrThearMan/graphene-django-query-optimizer/blob/9d58c3be48faa6a4551f06c20c3c712dff4f7068/query_optimizer/utils.py#L115)), so it would need to be somehow converted to an ORM expression. While this might be possible, it would certainly add to the cost of the query.

With these caveats laid out, I'll still try to fix the behavior.

**Comment by MrThearMan** [comment]
Ok, this should be fixed now. Note that using the `last` filter does now incur a performance penalty since it needs to calculate `totalCount` and use [calculate_slice_for_queryset](https://github.com/MrThearMan/graphene-django-query-optimizer/blob/aae03b2275ac38bc26a03207b54bf1dfab82959c/query_optimizer/utils.py#L176) to add aliases for start and stop indexes for each partition. Released in [0.2.9](https://github.com/MrThearMan/graphene-django-query-optimizer/releases/tag/v0.2.9).
